### ISSUE TITLE
[Accessibility]: Doc sidebar selection color can be hard to see

### ISSUE THREAD

**Issue by jmwoliver** [posit-dev/product-doc-theme#123]
**Describe the accessibility issue**
Selecting an item from the Docs sidebar table of contents on the left changes the selected item's color from black to dark blue. This can be a difficult change to see at a glance. Within a page, selecting an item in the right sidebar changes the color and bolds the item, making it somewhat easier to see. It would be nice if the left sidebar could match the selection colors and bolding like the right sidebar.

**Resolution**
Matching the color and bolding in the left sidebar like the right sidebar selection.

**Additional context**
This behavior can be observed here: https://packagemanager.posit.co/__docs__/admin/licensing.html#activation-options

<img width="4830" height="1896" alt="Image" src="https://github.com/user-attachments/assets/ec94d051-64e4-4d27-9986-236c12f2f2f6" />

**Comment by AshleyHenry15** [comment]
@jmwoliver 

In v8 of the theme, we changed the selected text sidebar to the following style:

<img width="991" height="469" alt="Image" src="https://github.com/user-attachments/assets/ee28ccde-0827-4975-a223-87350f4406f1" />

Let me know if that is an improvement and satisfies this issue/request

**Comment by jmwoliver** [comment]
I think that looks good! This issue can be closed as solved in v8. When will v8 be released?

**Comment by AshleyHenry15** [comment]
@jmwoliver - v8 is ready to use but it requires Quarto 1.8. Since most teams aren't comfortable with using quarto 1.8 until it is officially released, I will leave v8 in "pre release" until it officially rolls out


I also bumped the font weight a bit for both the sidebar and the TOC

**Comment by AshleyHenry15** [comment]
Footer shouldn’t be busted. I’ll take a look!

Ashley Henry - Lead Technical Writer

*(she/her/they)*
